### PR TITLE
Create VB Landing page

### LINKED ISSUE #16003: Update VB.NET landing page
The VB.NET landing page needs to conform to the new format. As part of this update, the following content changes should be made: 

- [ ] Highlight the "get started" journey
- [ ] Direct readers to the key content for VB.NET
- [ ] Highlight new and updated content.

This should be done by examining existing traffic. We need to highlight the most popular content, and the content we want to be more popular.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #16003 

Modeled after the other language landing pages, highlighting the most popular VB content.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/visual-basic/?branch=pr-en-us-16015)

### CODE REVIEW

**Comment by BillWagner** [comment]
@KathleenDollard What other content do you think should be highlighted here?

**Review by gewarren — COMMENTED** [review]
Looks like a good start!

File: docs/visual-basic/index.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+### YamlMime:Landing
+
+title: VB.NET documentation # < 60 chars
+summary: "VB is engineered for productively building type-safe and object-oriented apps. VB.NET continues the tradition of a fast and easy way to create .NET apps." # < 160 chars
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/visual-basic/index.yml]: Checking with @BethMassi

File: docs/visual-basic/index.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+### YamlMime:Landing
+
+title: VB.NET documentation # < 60 chars
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/visual-basic/index.yml]: All references should be *Visual Basic*

**Comment by BillWagner** [comment]
@gewarren @mairaw This is ready for final review.

**Comment by gewarren** [comment]
I think something weird happened with your force push. A bunch of my original suggestions were overwritten.

File: docs/toc.yml [hunk 3]
```diff
@@ -10,11 +10,11 @@
   href: desktop-wpf/
 - name: C# guide
   href: csharp/index.yml
-- name: F# guide
-  href: fsharp/
-- name: Visual Basic guide
-  href: visual-basic/
-- name: ML.NET guide
+- name: F# Guide
```
Comment (gewarren) [inline docs/toc.yml]:
```suggestion
- name: F# guide
```

File: docs/toc.yml [hunk 4]
```diff
@@ -10,11 +10,11 @@
...
+  href: fsharp/index.yml
+- name: Visual Basic Guide
```
Comment (gewarren) [inline docs/toc.yml]:
```suggestion
- name: Visual Basic guide
```

File: docs/toc.yml [hunk 5]
```diff
@@ -10,11 +10,11 @@
...
+  href: visual-basic/index.yml
+- name: ML.NET Guide
```
Comment (gewarren) [inline docs/toc.yml]:
```suggestion
- name: ML.NET guide
```

File: docs/visual-basic/index.yml [hunk 6]
```diff
@@ -0,0 +1,75 @@
+### YamlMime:Landing
+
+title: Visual Basic documentation # < 60 chars
+summary: "Visual Basic is engineered for productively building type-safe and object-oriented apps. Visual Basic continues the tradition of a fast and easy way to create .NET apps." # < 160 chars
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
summary: "Visual Basic is engineered to help you productively build type-safe and object-oriented apps. Visual Basic is a fast and easy way to create .NET apps." # < 160 chars
```

File: docs/visual-basic/index.yml [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+# Cards and links should be based on top customer tasks or top subjects
+# Start card title with a verb
+  # Card (optional)
+  - title: Get started
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Build a "Hello World" app with .NET Core
+            url: ../core/tutorials/vb-with-visual-studio.md
+          - text: Build a class library with .NET Core`
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: Build a class library with .NET Standard
```

File: docs/visual-basic/index.yml [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+            url: ../core/tutorials/vb-library-with-visual-studio.md
+          - text: Additional resources
+            url: getting-started/additional-resources.md
+          - text: Walkthroughs that demonstrate aspects of Visual Basic.
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: Walkthroughs
```

File: docs/visual-basic/index.yml [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+            url: programming-guide/concepts/index.md
+      - linkListType: concept
+        links:
+          - text: "Program structure and code conventions"
+            url: programming-guide/program-structure/program-structure-and-code-conventions.md
+          - text: "Develop apps with Visual Basic"
+            url: developing-apps/index.md
+          - text: "Program structure and code conventions"
+            url: programming-guide/program-structure/program-structure-and-code-conventions.md
+          - text: "Language features - LINQ, XML literals"
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: Language features
```

File: docs/visual-basic/index.yml [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+            url: programming-guide/language-features/index.md
+          - text: "COM Interop"
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: COM interop
```

File: docs/visual-basic/index.yml [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+            url: programming-guide/com-interop/index.md
+
+  - title: "Popular scenarios"
+    linkLists:
+      - linkListType: how-to-guide
+      # Awaiting link recommendations from product folks
+        links:
+          - text: Windows Forms Custom delegates and events
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: Windows Forms custom delegates and events
```

File: docs/visual-basic/index.yml [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+            url: https://social.technet.microsoft.com/wiki/contents/articles/53470.vb-net-windows-forms-custom-delegates-and-events.aspx
+          - text: Edit Database Table Values via WPF Datagrid
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: Edit database table values via WPF DataGrid
```

File: docs/visual-basic/index.yml [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+            url: https://social.msdn.microsoft.com/Forums/vstudio/10a3dd61-7dcc-48f3-a6e0-f7386d06f9f1/edit-database-table-values-via-wpf-datagrid-visual-basic?forum=wpf
+          - text: "Windows Forms ListView: Examining Detail view with SQL-Server"
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: "Windows Forms ListView: Examining detail view with SQL Server"
```

File: docs/visual-basic/index.yml [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+            url: https://social.technet.microsoft.com/wiki/contents/articles/53356.windows-forms-listview-examining-detail-view-with-sql-server.aspx
+          - text: "How to open excel 2016 workbook through OleDbConnection"
```
Comment (gewarren) [inline docs/visual-basic/index.yml]:
```suggestion
          - text: "How to open Excel 2016 workbook through OleDbConnection"
```

**Review by gewarren — COMMENTED** [review]
I re-added the suggestions I noticed from before.

**Comment by BillWagner** [comment]
@gewarren Hmm. Thanks for re-adding all those. I must have mis-handled something while rebasing to resolve conflicts. It should be correct now.

**Review by gewarren — APPROVED** [review]
Looks good!

**Pull request merged**